R/run-model.R

```diff
@@ -76,8 +89,8 @@ hintr_run_model <- function(data, options,
                  max_iter = options$max_iterations %||% 250,
                  progress = progress)
 
-  if(fit$convergence != 0 && !options$permissive) {
-    stop(paste("convergence error:", fit$message))
+  if(fit$convergence != 0) {
+    naomi_warning(paste("convergence error:", fit$message), "model_fit")
   }
```

Comment:
@jeffeaton do we want to totally remove the error thrown by false convergence (implemented here) or still have the option with permissive = false?

Reply:
I am happy to remove the error and fully convert it to a warning

Reply:
Should we also remove `permissive` option?

Reply:
I just did a quick search through the code -- the `permissive` option is not used anywhere else, so yes fine to remove. It's also not a big deal to keep it; it is pretty hidden away from user.